content/gotraining.slide

```diff
@@ -9,10 +9,229 @@ https://github.com/anz-bank/go-training
 
 * Introduction
 
-* Test
 
-# * Go history
 
-# * Why Go at ANZ
+* Agenda (11 - 15 February)
 
-# * Ping server Java Spring Boot vs Go
+- Introduction to Go
+- Go language fundamentals
+- Go standard library
+- Go tool chain
+- Third party Go libraries used at ANZ
+- Project work
+
+Find Slides at [[https://gotraining.appspot.com]] ([[https://github.com/anz-bank/go-slides][source]] on GitHub)
+
+
+* Day 1 - Schedule
+
+*Morning:* Introduction to Go
+
+- Overview and history
+- Demo
+
+*Afternoon:* Hello world and first PR
+
+- Interlude: towards continuous deployment (CD)
```

Comment:
might be worth defining what continuous deployment is

lots of people use "CD" to mean "continuous delivery", and often they talk about doing "continuous delivery" WITHOUT having anything resembling "continuous deployment"

Reply:
I've got a branch lined up for the next PR where I'm flashing these things out. I've added it there.